README.rst

```diff
@@ -1,22 +1,50 @@
 .. image:: https://img.shields.io/badge/github-repo-000.svg?logo=github&labelColor=gray&color=blue
-   :target: https://github.com/SecConNet/proof_of_concept
+   :target: https://github.com/SecConNet/mahiru
    :alt: GitHub Badge
 
-.. image:: https://github.com/SecConNet/proof_of_concept/workflows/Continuous%20Integration/badge.svg
-   :target: https://github.com/SecConNet/proof_of_concept/actions?query=workflow%3A%22Continuous+Integration%22
+.. image:: https://github.com/SecConNet/mahiru/workflows/Continuous%20Integration/badge.svg
+   :target: https://github.com/SecConNet/mahiru/actions?query=workflow%3A%22Continuous+Integration%22
    :alt: Continuous Integration Badge
 
-.. image:: https://img.shields.io/github/license/SecConNet/proof_of_concept
-   :target: https://github.com/SecConNet/proof_of_concept
+.. image:: https://img.shields.io/github/license/SecConNet/mahiru
+   :target: https://github.com/SecConNet/mahiru
    :alt: License Badge
 
-################################################################################
-Distributed Digital Data Marketplace Proof of Concept
-################################################################################
+#######################################
+Mahiru Data Exchange - Proof of Concept
+#######################################
 
-A proof of concept for a federated, policy-driven Digital Data Marketplace for
-the SecConNet project.
+A proof of concept for a federated, policy-driven data exchange for the
+SecConNet project.
 
+Mahiru is a design for a distributed data processing system which lets data and
+software owners share their data and software while keeping it under their
+control to any desired extent by defining policies. Users (or the applications
+they use) submit workflows expressing their desired data processing operation,
+and the system will determine automatically whether the workflow can be
+executed, and if so how.
+
+Mahiru's execution model is very flexible, and can support data downloads,
+compute-to-data, software-as-a-service, infrastructure-as-a-service, trusted
+third parties, and distributed machine learning, and it will automatically
+assemble any combination of these required to fulfil a request.
+
+Mahiru is a federated design, which means that participating parties can run a
+Mahiru site on their own hardware, on their own premises and fully under their
```

Comment:
Wow, really great description Lourens :) I think this is the exact right tone to 'advertise' mahiru.